license/license.go

```diff
@@ -198,6 +199,7 @@ func EncryptLicenseFields(l *License, c index.Content) error {
 
 	// empty the passphrase hash to avoid sending it back to the user
 	l.Encryption.UserKey.Value = nil
+	l.Encryption.UserKey.HexValue = ""
 
```

Comment:
Just wondering: is empty string treated the same as `nil`? (to ensure removal of `hex_value` from the serialized JSON)

Reply:
Yes, I checked it in the golang json doc and tested it.

Reply:
'omitempty' in the declaration makes the trick.